.goreleaser.common.yml

```diff
@@ -12,6 +12,7 @@ builds:
     ldflags:
       - -s -w
       - -X github.com/dagger/dagger/engine.Version={{.Env.ENGINE_VERSION}}
+      - -X github.com/dagger/dagger/engine.Tag={{.Env.ENGINE_VERSION}}
```

Comment:
Don't we want to set this to a different value, i.e. `WithEnvVariable("ENGINE_TAG", cli.Dagger.Version.TagString()` here https://github.com/sipsma/dagger/blob/2a9bb343e013194b9153739fe17a06c55282a841/dev/cli.go#L85

Reply:
Eek, yes.

Reply:
Hm, actually, the timestamps are an issue here :thinking: Because we publish the cli and engine in separate steps, the timestamps will be different, which is weird.

I guess we can get the timestamp in github actions, and pass it everywhere, but that's really annoying :cry:

Reply:
Fixed this in the last commit, by adding a new `ENGINE_TAG` env that's populated from the new `--tag` parameter. `dev` builds won't have this set, so we should error out - which makes sense, the current behavior is to try and pull a non-existent image.

Additionally, the timestamps in version numbers are an issue for the github multi-step release - this is resolved by just having a version pre-calculation step, which is messy, but eventually we'll shift all of this logic into dagger, so it'll hopefully all go away...